poetry.lock

```diff
@@ -7,9 +7,9 @@ optional = false
 python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
 
 [package.extras]
-dev = ["coverage[toml] (>=5.0.2)", "hypothesis", "pympler", "pytest (>=4.3.0)", "six", "mypy", "pytest-mypy-plugins", "zope.interface", "furo", "sphinx", "sphinx-notfound-page", "pre-commit", "cloudpickle"]
-docs = ["furo", "sphinx", "zope.interface", "sphinx-notfound-page"]
-tests = ["coverage[toml] (>=5.0.2)", "hypothesis", "pympler", "pytest (>=4.3.0)", "six", "mypy", "pytest-mypy-plugins", "zope.interface", "cloudpickle"]
+dev = ["coverage[toml] (>=5.0.2)", "hypothesis", "pympler", "pytest (>=4.3.0)", "six", "mypy", "pytest-mypy-plugins", "zope-interface", "furo", "sphinx", "sphinx-notfound-page", "pre-commit", "cloudpickle"]
```

Comment:
I'm not sure why it decided to update the zope dependencies as well?

Reply:
How did you do the lock---`poetry lock --no-update`?

Reply:
Yup

Reply:
And what's your poetry version, for completeness?

Reply:
It looks like poetry 1.2 uses dashes rather than full stops in package names. I think they are also explicitly requiring setuptools rather than implicitly assuming it's always in the virtualenv.

I'd guess that the poetry version we use in CI (1.1.12 or maybe 1.1.13) has been thrown off by full stop -> dash rename. Let me see if I can generate a lock file for you on 1.1.x.

Reply:
Looks good now! I assume I'm using a beta of 1.2 following some tests I did last time I had a dependency issue and just forgot to roll back. Thanks for taking a look and fixing!